opentelemetry-sdk/Cargo.toml

```
@@ -49,9 +49,10 @@ logs = ["opentelemetry/logs", "serde_json"]
 spec_unstable_logs_enabled = ["logs", "opentelemetry/spec_unstable_logs_enabled"]
 metrics = ["opentelemetry/metrics", "glob", "async-trait"]
 testing = ["opentelemetry/testing", "trace", "metrics", "logs", "rt-async-std", "rt-tokio", "rt-tokio-current-thread", "tokio/macros", "tokio/rt-multi-thread"]
-rt-tokio = ["tokio", "tokio-stream"]
-rt-tokio-current-thread = ["tokio", "tokio-stream"]
-rt-async-std = ["async-std"]
+experimental_runtime = []
```

```suggestion
experimental_async_runtime = []
```

async_runtime is more explicit?